src/tests/ftest/io/nvme_io.py

```diff
@@ -67,6 +69,20 @@ def setUp(self):
         self.hostlist = self.params.get("servers", '/run/hosts/*')
         self.hostfile = write_host_file.write_host_file(self.hostlist,
                                                         self.workdir)
+        self.hostlist_clients = self.params.get("clients", '/run/hosts/*')
```

Comment:
I realize this is probably another PR, but do we standardize on server/client yaml file definitions? According to a quick search I see:
- 3 yaml files with /run/hosts/test_[servers|clients]
- 1 yaml file with /run/hosts/test_servers + /run/hosts/clients
- 3 yaml files with /run/hosts/test_machines/test_[servers|clients]
- 2 yaml files with /run/hosts/[servers|clients]
- 1 yaml file with /run/hosts/test_machines/daos_servers
To use apricot to its full potential it would be nice to have a standard.